.github/workflows/linux-ci-helper.sh

```diff
@@ -369,6 +362,7 @@ elif [ "${CONTAINER_FULLNAME}" = "opensuse/leap:15" ]; then
         openssl
         openssl-devel
         procps
+        python3
```

Comment:
`clang-tools` on OpenSuSe includes some Python scripts and thus previously implicitly required `python3`.